eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -243,4 +254,11 @@ private function executeCommand(OutputInterface $output, $cmd, $timeout = 300)
             throw new \RuntimeException(sprintf('An error occurred when executing the "%s" command.', escapeshellarg($cmd)));
         }
     }
+
+    private function getCurrentConnectionName(): string
+    {
+        $repositoryConfig = $this->repositoryConfigurationProvider->getRepositoryConfig();
+
+        return $repositoryConfig[self::REPOSITORY_STORAGE][self::REPOSITORY_CONNECTION];
```

Comment:
> Instead of the constants and the private method here. Wouldn't it be more readable and simplify the code if we simply expose a `$this->repositoryConfigurationProvider->getConnectionName();` method.
> 
> It's not like this is the only place that we will have to deal with this afaik :)

The diff above is definitely internal to the Repository configuration, so I agree. I'd only adjust naming to `getStorageConnectionName` as the Repo config is not limited just to storage.

On the other hand, not so sure if knowing a connection name is a responsibility of configuration provider, but I still think here it's a first good step.

In the future I would see some domain (value) objects which represent the configuration so we can do:
```php
$this->repositoryConfigurationProvider->getStorageConfiguration()->getConnectionName();
```
or
```php
$this->repositoryConfigurationProvider->getRepositoryConfiguration()->getStorageConfiguration()->getConnectionName()
```
or even using `Repository` "container"
```php
$this->repository->getStorage()->getConnectionName()
```
_(but the last one might be technically challenging due to cyclic dependencies or other issues)_

For now indeed I'd go with
```php
$this->repositoryConfigurationProvider->getStorageConnectionName();
```

cc @adamwojs